packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md

```diff
@@ -138,11 +146,11 @@ This example shows the JSON body of a PUT request to an existing OpenID Connect
 }
 ```
 
-To enable consent for a scope, you need to [update the appropriate scope](/docs/reference/api/authorization-servers/#update-a-scope) by setting the `consent` property for the scope from `IMPLICIT` (the default) to `REQUIRED`. You can also set the `consent` property for the scope to `FLEXIBLE`. See the [Authorization Servers API](/docs/reference/api/authorization-servers/#scope-properties) for more information on this value.
-
 ### Update Scope consent
 
-This example shows the JSON body for a PUT request to the default Custom Authorization Server (`https://${yourOktaDomain}/api/v1/authorizationServers/${authorizationServerId}/scopes/${scopeId}`) to update the `phone` scope. You need the following information for the request:
+To enable consent for a scope, you need to [update the appropriate scope](/docs/reference/api/authorization-servers/#update-a-scope) by updating the `consent` property for the scope from `IMPLICIT` (the default) to `REQUIRED`. You can also set the `consent` property for the scope to `FLEXIBLE` or `OPTIONAL`. See the [Authorization Servers API](/docs/reference/api/authorization-servers/#scope-properties).
```

Comment:
You can set the `consent` property for the scope to IMPLICIT, REQUIRED or FLEXIBLE
To make a scope optional, one needs to include new field in the request "optional" : true,
example
```
{
    "consent": "REQUIRED",
    "optional" : true,
    "metadataPublish": "NO_CLIENTS",
    "name": "tst1",
    "displayName": "as2-scp6",
    "description": "as2-scp6 desc"
}
```

Reply:
you can see the mapping here
https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2498241876/Technical+Design+for+Granular+Consent+Eng+1-Pager#Database-saasure_prod